src/facade/dragonfly_connection.cc

```diff
@@ -585,10 +577,15 @@ void Connection::ConnectionFlow(FiberSocketBase* peer) {
   // After the client disconnected.
   cc_->conn_closing = true;  // Signal dispatch to close.
   evc_.notify();
-  VLOG(1) << "Before dispatch_fb.join()";
-  if (dispatch_fb_.IsJoinable())
-    dispatch_fb_.Join();
-  VLOG(1) << "After dispatch_fb.join()";
+
+  {
```

Comment:
what is the reason for block scope change?

Reply:
Pure stylistic reasons 😄 But I reverted it once it got shorter with the JoinIfNeeded